apps/ExampleApp/app.config.ts

```diff
@@ -17,6 +16,6 @@ module.exports = ({ config }: ConfigContext): Partial<ExpoConfig> => {
   const existingPlugins = config.plugins ?? []
   return {
     ...config,
-    plugins: [...existingPlugins, withSplashScreen, withFlipperDisabled],
+    plugins: [...existingPlugins, withFlipperDisabled],
```

Comment:
flipper is disabled by default, we could probably remove the plugin listed here, the files associated with it and then remove the `"flipper": false,` for the ios part in `app.json` under `expo-build-properties` plugin